src/sagemaker/jumpstart/curated_hub/curated_hub.py

```diff
@@ -301,4 +332,16 @@ def _sync_public_model_to_hub(self, model: Dict[str, str]):
             hub_content_description="",
             hub_content_markdown="",
             hub_content_search_keywords=[],
+            tags=tags,
+        )
+
+    def _fetch_studio_specs(self, model_specs: JumpStartModelSpecs) -> Dict[str, Any]:
+        """Fetches StudioSpecs given a models' SDK Specs."""
+        model_id = model_specs.model_id
+        model_version = model_specs.version
+
+        key = f"studio_models/{model_id}/studio_specs_v{model_version}.json"
```

Comment:
can u store this as a constant somewhere and ensure this function gets unit tested? this seems like a delicate piece of code